src/Microsoft.ML.TorchSharp/NasBert/TextClassificationTrainer.cs

```diff
@@ -668,31 +671,32 @@ public sealed class TextClassificationTransformer : RowToRowTransformerBase
 
         private readonly Device _device;
         private readonly TextClassificationModel _model;
-        private readonly Vocabulary _vocabulary;
         private readonly TextClassificationTrainer.Options _options;
 
         private readonly string _predictedLabelColumnName;
         private readonly string _scoreColumnName;
 
         public readonly SchemaShape.Column SentenceColumn;
         public readonly SchemaShape.Column SentenceColumn2;
-        public readonly SchemaShape.Column LabelColumn;
+        public readonly DataViewSchema.DetachedColumn LabelColumn;
```

Comment:
what's the difference between `SchemaShape.Column` and `DataViewSchema.DetachedColumn`

Reply:
`SchemaShape.Column` is attached to an actual `IDataView`. A `DetachedColumn` has all the same info but is not attached to any specific `IDataView`.